### PR TITLE
gerbv 2.8.2

### PR DESCRIPTION
**Pull request by SMillerDev**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/gerbv.rb [hunk 1]
```diff
@@ -1,16 +1,10 @@
 class Gerbv < Formula
   desc "Gerber (RS-274X) viewer"
-  homepage "http://gerbv.gpleda.org/"
-  # 2.6.1 is the latest official stable release but it is very buggy and incomplete
-  url "https://downloads.sourceforge.net/project/gerbv/gerbv/gerbv-2.7.0/gerbv-2.7.0.tar.gz"
-  sha256 "c5ee808c4230ce6be3ad10ab63c547098386d43022704de25ddb9378e62053b4"
+  homepage "https://gerbv.github.io/"
+  url "https://github.com/gerbv/gerbv/archive/refs/tags/v2.8.2.tar.gz"
+  sha256 "588ec0bf86994ff5fff50716f9644562c5b74332e19ada5ec8af2256f040c0a0"
   license "GPL-2.0"
```
Comment (Moisan) [inline Formula/gerbv.rb:6]: Upstream header includes 
```
 * This program is free software; you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation; either version 2 of the License, or
 * (at your option) any later version.
 ```

```suggestion
  license "GPL-2.0-or-later"
```

File: Formula/gerbv.rb [hunk 2]
```diff
@@ -21,16 +15,26 @@ class Gerbv < Formula
...
 
   def install
-    ENV.append "CPPFLAGS", "-DQUARTZ"
-    system "./configure", "--disable-debug",
+    ENV.append "CPPFLAGS", "-DQUARTZ" if OS.mac?
+    inreplace "autogen.sh", "libtool", "glibtool"
+
+    # Disable commit reference in include dir
+    inreplace "utils/git-version-gen.sh", 'RELEASE_COMMIT=`"${GIT}" rev-parse HEAD`', "RELEASE_COMMIT=\"\""
+    inreplace "utils/git-version-gen.sh", "${PREFIX}~", "${PREFIX}"
```
Comment (carlocab) [inline Formula/gerbv.rb:31]:
```suggestion
    inreplace "utils/git-version-gen.sh", do |s|
      s.gsub! 'RELEASE_COMMIT=`"${GIT}" rev-parse HEAD`', "RELEASE_COMMIT=\"\""
      s.gsub! "${PREFIX}~", "${PREFIX}"
    end
```
Why is this needed?

Comment (SMillerDev) [reply Formula/gerbv.rb:31]: To stop it from trying to find commit hashes in a tarbal.

**Review by carlocab — APPROVED** [review]
